test/Syntax/deserialize.swift

```diff
@@ -0,0 +1,402 @@
+// RUN: %empty-directory(%t)
+// RUN: %swift-syntax-test -serialize-raw-tree -input-source-filename %s > %t/serialized.json
+// RUN: %swift-syntax-test -deserialize-raw-tree -input-source-filename %t/serialized.json > %t/output.swift
+// RUN: diff -u %s %t/output.swift
```

Comment:
Can we add these RUN lines to the other test `round_trip_parse_gen.swift` instead of duplicating the whole file? We can get some free test case when others extending `round_trip_parse_gen.swift` 😄

Reply:
Could you please tell me how to use testsuite to trigger these RUN lines?
I used `util/build-script --test` but this is so slow. ☹️

Reply:
hmm, there's no easy way to run a specific test from the script. Sorry.. I think this commit is good as it is and we can always improve later 👍